frontend/institution/managementMembersController.js

```diff
@@ -234,6 +243,26 @@
             return promise;
         };
 
+        manageMemberCtrl.canAddAnotherEmailField = function canAddAnotherEmailField() {
+            if (_.size(manageMemberCtrl.emails) < MAX_EMAILS) {
+                addAnotherField()
+            };
+        };
+
+        function addAnotherField() {
```

Comment:
Why is another? Why not just addField?

Reply:
Another field seemed more logical to me. addField gives me the idea that there is no other fields before that. But I can change it